scripts/build-stubs.sh

```diff
@@ -1,13 +1,38 @@
 #!/usr/bin/env bash
 
-git clone --depth 1 --branch main https://github.com/adafruit/circuitpython.git
-cd circuitpython
-git submodule init
-git submodule update extmod/ulab
-pip3 install -r docs/requirements.txt
-make stubs
-mv circuitpython-stubs ../stubs
-cd ..
-pip3 install -r requirements.txt
-python3 ./scripts/build_stubs.py
-rm -rf stubs/board
\ No newline at end of file
+(
+    # Current dir should be the root of the repo
+    cd $(dirname $0)/..
```

Comment:
I actually run this from the root of the repo `./scripts/build-stubs.sh`. So does the azure pipeline, so that would have to be adjusted as well. I may move to github actions soon anyway, so that may be a moot point.

Reply:
Yes, that's what I figured out. This just makes it not matter - the parens open a sub-shell, and that line changes directory to the root of the repo. Just a personal habit/tweak, I'm not fond of scripts where you can run them from the "wrong" directory and have them fail. So, this will now always act as if it was run from the correct place.